ArmPkg/Library/ArmMmuLib/AArch64/ArmMmuLibCore.c

```diff
@@ -568,6 +568,8 @@ ArmConfigureMmu (
   UINT64      TCR;
   EFI_STATUS  Status;
 
+  ASSERT (ArmReadCurrentEL () < AARCH64_EL3);
+

```

Comment:
Should we also return EFI_UNSUPPORTED? The existing code had this behaviour, however I acknowledge that the return value is not being checked.

Reply:
Yes that makes sense.